todo_app/app.py

```diff
@@ -4,219 +4,166 @@
-
-	response = requests.request(
-		"PUT",
-		url,
-		headers=headers,
-		params=query
-	)
+	toDoId = os.environ['TOID']
+	doingId = os.environ['DOINGID']
+	doneId = os.environ['DONE']
+	def getItems(idList):
+		url = f"https://api.trello.com/1/lists/{idList}/cards"
+		headers = {
+			"Accept": "application/json"
+		}
+		query = {
+	    	'id' : idList,
+			'key': os.environ['KEY'],
+			'token' : os.environ['TOKEN'],
+			'fields': 'all'
+		}
+		response = requests.get(
+			
+			url,
+			headers=headers,
+			params=query
+		)
 
-def returnCardToDo(i):
+		return response
+	def newToDoCard(name):
+		url = "https://api.trello.com/1/cards"
+		query = {
+			'key': os.environ['KEY'],
+			'token' : os.environ['TOKEN'],
+			'idList': toDoId,
+			'name': name
+		}
+		response = requests.request(
+			"POST",
+			url,
+			params=query
+		)
+	def updateCardToDone(i):
+		url = f"https://api.trello.com/1/cards/{i}"
+		headers = {
+			"Accept": "application/json"
+		}
+		query = {
+	    	'id': i,
+			'key': os.environ['KEY'],
+			'token' : os.environ['TOKEN'],
+			'idList': doneId,
+		}
+		response = requests.request(
+			"PUT",
+			url,
+			headers=headers,
+			params=query
+		)
+	def returnCardToDo(i):
 		url = f"https://api.trello.com/1/cards/{i}"
-
 		headers = {
 			"Accept": "application/json"
 		}
-
 		query = {
 			'id': i,
 			'key': os.environ['KEY'],
 			'token' : os.environ['TOKEN'],
 			'idBoard': '6005828032dafa5707bf5dc3',
-			'idList': '6005828032dafa5707bf5dc5'
-		
+			'idList': toDoId,
 		}
-
 		response = requests.request(
 			"PUT",
 			url,
 			headers=headers,
 			params=query
-	)
-
-
-
-@app.route('/')
-
-def index():
- response = getToDo()
- responseDoing = getDoing()
- responseDone = getDone()
- jsonResponse = response.json()
- jsonResponseDoing = responseDoing.json()
- jsonResponseDone = responseDone.json()
- number = len(jsonResponse)
- numberTwo = len(jsonResponseDoing)
- numberThree = len(jsonResponseDone)
- 
- my_objects = []
- 
- for listNumber in range(number):
+		)
+
+	@app.route('/')
+	def index():
+		idList = toDoId
+		responseTodo= getItems(idList)
+		jsonResponse = responseTodo.json()
+		idListDoing = doingId
+		responseDoing = getItems(idListDoing)
+		jsonResponseDoing = responseDoing.json()
+		idListDone = doneId
+		responseDone = getItems(idListDone)
+		jsonResponseDone = responseDone.json()
+		number = len(jsonResponse)
+		number = len(jsonResponse)
+		numberTwo = len(jsonResponseDoing)
+		numberThree = len(jsonResponseDone)
+
+
+		my_objects = []
+		doing_objects =[]
+		done_objects = []
+
+		for listNumber in range(number):		
+			my_objects.append(ToDo(jsonResponse[listNumber]['id'],jsonResponse[listNumber]['name'],jsonResponse[listNumber]['dateLastActivity']))
+		for listNumberDoing in range(numberTwo):		
+			doing_objects.append(ToDo(jsonResponseDoing[listNumberDoing]['id'],jsonResponseDoing[listNumberDoing]['name'],jsonResponseDoing[listNumberDoing]['dateLastActivity']))
+		for listNumberDone in range(numberThree):		
```

Comment:
It would be clearer to inline these "number" variables, i.e.
```suggestion
		for listNumberDone in range(len(jsonResponseDone)):		
```

Reply:
Updated,thank you